core/gather/gatherers/script-elements.js

```diff
@@ -65,7 +65,7 @@ class ScriptElements extends BaseGatherer {
 
     const scriptRecords = networkRecords
       .filter(record => record.resourceType === NetworkRequest.TYPES.Script)
-      .filter(record => !record.isOutOfProcessIframe);
+      .filter(record => record.sessionTargetType === 'page');
```

Comment:
There are several places `isOutOfProcessIframe` is used where it could make more sense to use a `sessionTargetType === 'page'` now that we have worker requests as well. This is just the only place that caused a test failure.

Reply:
Update: I changed the other usages of `isOutOfProcessIframe` too. We don't use `isOutOfProcessIframe` anywhere now.